### ISSUE TITLE
RTSP TCP TRANSPORT

### ISSUE THREAD

**Issue by StavrosAnastopoulos** [k-yle/rtsp-relay#82]
-rtsp_transport tcp flag needs to be inserted at the start of the ffmpeg command in order to work as stated https://stackoverflow.com/a/23302601

I had to download your repo locally and change index.js as following to force tcp connection

```
...
start({ url, additionalFlags = [] }) {
    if (this.verbose) console.log('[rtsp-relay] Creating brand new stream');

    this.stream = spawn(
      ffmpegPath,
      [
        '-rtsp_transport',
        'tcp',
        '-i',
        url,
        '-f', // force format
        'mpegts',
        '-codec:v', // specify video codec (MPEG1 required for jsmpeg)
        'mpeg1video',
        '-r',
        '30', // 30 fps. any lower and the client can't decode it
        ...additionalFlags,
        '-',
      ],
      { detached: false },
    );
...
```

**Comment by k-yle** [comment]
oh that's annoying... will fix this when I get a chance

**Comment by k-yle** [comment]
Hi, this is fixed in `v1.5.0` 

instead of `additionalFlags: ["-rtsp_transport", "tcp"]`, use `transport: "tcp"`. 

I have updated the README and there will be a warning in the console if you specify `-rtsp_transport` in `additionalFlags`

**Comment by sh2** [comment]
I think 'proxy' function also needs 'transport' option.
https://github.com/k-yle/rtsp-relay/blob/82a2cae984d7cec04595ac7d832ad496c5f27082/index.js#L150

and here.
https://github.com/k-yle/rtsp-relay/blob/82a2cae984d7cec04595ac7d832ad496c5f27082/index.js#L169

**Comment by k-yle** [comment]
@sh2 👍 fixed in `v1.5.1`
